src/genie/libs/parser/iosxe/tests/ShowIpSlaSummary/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,46 @@
+expected_output = {
+      'id': {
+            '1': {'probe_status': 'active',
+                  'destination': '123.23.213.32',
+                  'last_run_seconds_ago': '21',
+                  'return_code': 'OK',
+                  'rtt_stats_mseconds': '44',
+                  'type': 'tcp-connect'},
+            '2': {'probe_status': 'active',
+                  'destination': '11.121.2.123',
+                  'last_run_seconds_ago': '7',
+                  'return_code': 'OK',
+                  'rtt_stats_mseconds': '79',
+                  'type': 'dns'},
+            '3': {'probe_status': 'active',
+                  'destination': '121.32.11.1',
+                  'last_run_seconds_ago': '54',
+                  'return_code': 'OK',
+                  'rtt_stats_mseconds': '1',
+                  'type': 'udp-jitter'},
+            '4': {'probe_status': 'active',
+                  'destination': '12.223.33.3',
+                  'last_run_seconds_ago': '15',
+                  'return_code': 'OK',
+                  'rtt_stats_mseconds': '1',
+                  'type': 'udp-jitter'},
+            '5': {'probe_status': 'active',
+                  'destination': '13.132.32.2',
+                  'last_run_seconds_ago': '8',
+                  'return_code': 'OK',
+                  'rtt_stats_mseconds': '1',
+                  'type': 'udp-jitter'},
+            '6': {'probe_status': 'active',
+                  'destination': '11.311.31.2',
+                  'last_run_seconds_ago': '40',
```

Comment:
sorry just noticed. the value can be integer.
```suggestion
                  'last_run_seconds_ago': 40,
```